db.go

```diff
@@ -0,0 +1,30 @@
+package testy
+
+import (
+	"context"
+	"errors"
+	"fmt"
+)
+
+var ErrNoDB = errors.New("no DB set")
+
+// DB is the interface for something which can save and retrieve test reports.
```

Comment:
looks like it can only save but not retrieve

Reply:
> This interface will be expanded shortly to enumerate and retrieve test results.